src/SerialFirmata.h

```diff
@@ -23,6 +23,10 @@
 #include <SoftwareSerial.h>
 #endif
 
+//uncomment SERIAL_STORE_AND_FORWARD to collect bytes received by serial port until the receive buffer 
+// gets filled or a data gap is detected to avoid forwarding single bytes at baud rates below 50000
+//#define SERIAL_STORE_AND_FORWARD
```

Comment:
Both are not perfect. There is always a possible confusion with the host link. If you prefer the latter I'm in, but here is another suggestions: ```SERIAL_PORT_RX_BUFFERING```.